src/test/demultiplexer_node_test.cpp

```diff
@@ -126,7 +129,10 @@ TEST(DemultiplexerTest, DemultiplyShouldReturnErrorWhenWrongOutputDimensions) {
     SessionResults sessionResults;
     session_key_t sessionKey = meta.getSessionKey();
     // perform test
-    auto status = demultiplexerNode.fetchResults(sessionKey, sessionResults);
+    ExecutionContext context(
```

Comment:
What do you think about having global default context in test_utils?

Reply:
Added global context for tests